src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -155,8 +160,10 @@ public bool TryUnparse(StringBuilder sb)
             }
         }
 
-        private sealed class ColInfoEx
+        public sealed class ColumnInfo
```

Comment:
Why not combine this with transformInfo? You can just add this public constructor and change the name to columnInfo, while keeping the other methods internal/private.

Reply:
I don't see any reason why you can't delete TransformInfo class and use ColumnInfo instead.

---
In reply to: [231632966](https://github.com/dotnet/machinelearning/pull/1410#discussion_r231632966) [](ancestors = 231632966)

Reply:
fixed

---
In reply to: [232104746](https://github.com/dotnet/machinelearning/pull/1410#discussion_r232104746) [](ancestors = 232104746,231632966)